web/public/locales/en.json

```diff
@@ -1814,6 +1814,18 @@
     "VN": {
       "zoneName": "Vietnam"
     },
+    "VN-C": {
+      "countryName": "Vietnam",
+      "zoneName": "Central Vietnam"
+    },
+    "VN-N": {
+      "countryName": "Vietnam",
+      "zoneName": "Southern Vietnam"
```

Comment:
Surly this should be Northern Vietnam? 😉

Reply:
Whoops, good catch - sorry North! 😅